matchbox-server/src/main/java/ch/ahdis/matchbox/validation/matchspark/LLMConnector.java

```diff
@@ -120,8 +130,13 @@ public String interpretWithMatchbox(String resource, String operationOutcome) {
             saveStringToFile(requestBody);
 
             chatMemory.add(UserMessage.from(requestBody));
-            ChatResponse response = model.chat(chatMemory.messages());
-            
+            ChatRequest request = ChatRequest.builder()
+                    .messages(chatMemory.messages())
+                    .build();
+            ChatResponse response = model.chat(request);
+
+            System.out.println("Input Tokens: " + response.tokenUsage().inputTokenCount());
+            System.out.println("Output Tokens: " + response.tokenUsage().outputTokenCount());
```

Comment:
No direct calls on `System.out` in the code (except for debugging purposes), you should use the slf4j logger instead